docs/src/Groups/pcgroup.md

```diff
@@ -92,28 +92,30 @@ and let Oscar compute a pc presentation for it.
 
 ```jldoctest
 julia> g = symmetric_group(4)
-Sym(4)
+Symmetric group of degree 4 and order 24 with 2 generators
+  (1,2,3,4)
+  (1,2)
 
 julia> iso = isomorphism(PcGroup, g);
 
 julia> h = codomain(iso)
-Pc group of order 24
+Pc group of order 24 with 4 generators [f1, f2, f3, f4]
 ```
 
 For certain series of groups, one can
 [create the members as pc groups](@ref "Series of polycyclic groups").
 
 ```jldoctest
 julia> dihedral_group(8)
-Pc group of order 8
+Pc group of order 8 with 3 generators [f1, f2, f3]
```

Comment:
I experimented further with printing the generators of (sub) fp-groups and (sub) pc-groups and arrived at this, inspired by how we print multivariate polynomial rings.